docs/spam_checker.md

```diff
@@ -31,6 +32,10 @@ are documented in the `synapse.events.spamcheck.SpamChecker` class.
 The `ModuleApi` class provides a way for the custom spam checker class to
 call back into the homeserver internals.
 
+Additionally, a `parse_config` method is mandatory and receives the plugin config
+dictionary. After possible sanitizing, It must return a dictionary which
+will be passed to `__init__` later.
```

Comment:
This doesn't need to return a dictionary, but any object which can go into the `__init__`, I think.

```suggestion
dictionary. After parsing, It must return an object which will be
passed to `__init__` later.
```